src/styles/lists.styl

```diff
@@ -175,6 +175,23 @@ heavyShadow = 0 4px 24px 0 rgba(50, 54, 63, .08)
                 background none
                 border 0
 
+.empty-services-list-tip
+    box-sizing border-box
+    width 640px
+    max-width 100%
+    margin auto
+    padding 1rem 2rem
+
+    +tiny-screen()
+        padding 0 1.5rem 1rem 3rem
+
+.empty-services-list-text
+    font-style italic
+    padding-left 1rem
+
+    +tiny-screen()
+        padding-left .5rem
+
```

Comment:
It seems that in this repo, we do not use the suit CSS convention that we follow (loosely) across our repos (https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md). Is there a good reason not to folllow it ? do you think we should use it for new code ?

Reply:
I'l have a proper look at it, but it's probably just me going with the existing convention I see in the code. And yeah def. makes sense to follow our normal conventions for new code 👍